app/scripts/lib/origin-check.js

```diff
@@ -77,7 +80,11 @@ define([
         .fail(function (err) {
           if (/Timed out/.test(String(err))) {
             // a timeout is A-OK, no allowed origins are listening.
+            Raven.captureException(new Error('Origin check timed out'));
+
             return null;
```

Comment:
There seems to be a logical error in the original code, like it was supposed to re-throw the error if the conditional wasn't satisfied. Otherwise, this return statement is a no-op.